airbyte-integrations/connectors/destination-iceberg/src/main/resources/spec.json

```diff
@@ -154,6 +154,28 @@
                 "order": 3
               }
             }
+          },
+          {
+            "title": "GlueCatalog",
```

Comment:
doublechecking: this means we're reusing the same creds as the s3 storage_config?

(what happens if someone tries to use server-managed storage + glue?)

Reply:
💥  won't work.  Needs some refactor of config blocks but didn't do since this is oss only new catalog.